### DISCUSSION TITLE
How I can build a .jar of my bot?

### DISCUSSION THREAD

**Discussion by jvramos-py** [kordlib/kord#837] [Help] [answered]
I'm trying to upload my bot at cloud, but when I try to build with the command:
`kotlinc -include-runtime {package} -d {package}.jar`

They returns that all imports, methods, etc. of Kord are unresolved reference, how I can fix that?

**Comment by HopeBaron** [comment, marked as answer]
Hello, easiest way you could solve this is to use the shadowjar gradle plugin and create a fat jar.
or you can lookup how to create a fatjar

**Reply by jvramos-py** [reply]
Thanks, I Will search about!

**Reply by DRSchlaubi** [reply]
However I would highly recommend not to use fatJars as their not using the class path as intended, it's highly recommended to use the Gradle application plugin
